apps/prisma-next-idb-kanban-example/src/routes/layout.css

```
@@ -68,6 +93,10 @@
 
   body {
     @apply bg-background text-foreground min-h-svh antialiased;
+    background-image:
+      linear-gradient(180deg, var(--surface-wash), transparent 22rem),
+      linear-gradient(0deg, var(--surface-lift), transparent 36rem);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)